python/doc/examples/functional_modeling/field_functions/plot_logistic_growth_model.py

```diff
@@ -20,6 +20,7 @@
 from matplotlib import pylab as plt
 
 ot.Log.Show(ot.Log.NONE)
+ot.RandomGenerator.SetSeed(0)
```

Comment:
This line is unnecessary.
```suggestion
```

Reply:
no this is ok in examples because they are run in the same process